kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -79,6 +81,9 @@
 public class XStreamMarshaller
         implements Marshaller {
 
+    public static final String SYSTEM_XSTREAM_ENABLED_PACKAGES = "org.kie.server.xstream.enabled.packages";
```

Comment:
can we move this system property to KieServerConstants where all other system properties are defined?